test/audits/bad-aria-attribute-test.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2014 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+(function() {
+    module('BadAriaAttribute');
+    var rule = axs.AuditRules.getRule('badAriaAttribute');
+
+    test('Element with role and global, supported and required attributes', function() {
+        var fixture = document.getElementById('qunit-fixture');
+        var widget = fixture.appendChild(document.createElement('div'));
+        var expected = { elements: [], result: axs.constants.AuditResult.PASS };
```

Comment:
As above: it would be good to move this below, to just above the assert, and add a comment explaining why this is expected.